data/abilities.ts

```diff
@@ -3170,7 +3170,7 @@ export const BattleAbilities: {[abilityid: string]: AbilityData} = {
 		shortDesc: "This Pokemon's attacks with recoil or crash damage have 1.2x power; not Struggle.",
 		onBasePowerPriority: 23,
 		onBasePower(basePower, attacker, defender, move) {
-			if (move.recoil || move.hasCustomRecoil) {
+			if (Array.isArray(move.recoil) || move.recoil === 'crash') {
```

Comment:
`Array.isArray` checks are not super pretty, this could be turned into a more pleasantly named helper type guard that does the check and return `is [number, number]`, but I couldnt think of a good name for 'normal' recoil

Reply:
`typeof move.recoil !== 'string'` would do the same thing. But I'm fine with `Array.isArray` too.